code/game/objects/items/toys.dm

```diff
@@ -547,70 +600,510 @@
+	desc = "4/13"
 	icon_state = "gygaxtoy"
+	max_combat_health = 5
+	special_attack_type = SPECIAL_ATTACK_UTILITY
+	special_attack_cry = "SUPER SERVOS"
 
 /obj/item/toy/prize/durand
 	name = "toy Durand"
-	desc = "Mini-Mecha action figure! Collect them all! 5/13."
+	desc = "5/13"
 	icon_state = "durandtoy"
+	max_combat_health = 6
+	special_attack_type = SPECIAL_ATTACK_HEAL
+	special_attack_cry = "SHIELD OF PROTECTION"
 
 /obj/item/toy/prize/honk
 	name = "toy H.O.N.K."
-	desc = "Mini-Mecha action figure! Collect them all! 6/13."
+	desc = "6/13"
 	icon_state = "honktoy"
+	max_combat_health = 4
+	special_attack_type = SPECIAL_ATTACK_OTHER
+	special_attack_type_message = "puts the opposing mech's special move on cooldown and heals this mech."
+	special_attack_cry = "MEGA HORN"
+
+/obj/item/toy/prize/honk/super_special_attack(obj/item/toy/prize/victim) 
+	playsound(src, 'sound/machines/honkbot_evil_laugh.ogg', 20, TRUE)
+	victim.special_attack_cooldown = 3 //Adds cooldown to the other mech and gives a minor self heal
+	combat_health++
 
 /obj/item/toy/prize/marauder
 	name = "toy Marauder"
-	desc = "Mini-Mecha action figure! Collect them all! 7/13."
+	desc = "7/13"
 	icon_state = "maraudertoy"
+	max_combat_health = 7
+	special_attack_type = SPECIAL_ATTACK_DAMAGE
+	special_attack_cry = "BEAM BLAST"
 
 /obj/item/toy/prize/seraph
 	name = "toy Seraph"
-	desc = "Mini-Mecha action figure! Collect them all! 8/13."
+	desc = "8/13"
 	icon_state = "seraphtoy"
+	max_combat_health = 8
+	special_attack_type = SPECIAL_ATTACK_DAMAGE
+	special_attack_cry = "ROCKET BARRAGE"
 
 /obj/item/toy/prize/mauler
 	name = "toy Mauler"
-	desc = "Mini-Mecha action figure! Collect them all! 9/13."
+	desc = "9/13"
 	icon_state = "maulertoy"
+	max_combat_health = 8
+	special_attack_type = SPECIAL_ATTACK_DAMAGE
+	special_attack_cry = "BULLET STORM"
 
 /obj/item/toy/prize/odysseus
 	name = "toy Odysseus"
-	desc = "Mini-Mecha action figure! Collect them all! 10/13."
+	desc = "10/13"
 	icon_state = "odysseustoy"
+	max_combat_health = 4
+	special_attack_type = SPECIAL_ATTACK_HEAL
+	special_attack_cry = "MECHA BEAM"
 
 /obj/item/toy/prize/phazon
 	name = "toy Phazon"
-	desc = "Mini-Mecha action figure! Collect them all! 11/13."
+	desc = "11/13"
 	icon_state = "phazontoy"
+	max_combat_health = 6
+	special_attack_type = SPECIAL_ATTACK_UTILITY
+	special_attack_cry = "NO-CLIP"
 
 /obj/item/toy/prize/reticence
 	name = "toy Reticence"
-	desc = "Mini-Mecha action figure! Collect them all! 12/13."
+	desc = "12/13"
 	icon_state = "reticencetoy"
-	quiet = 1
+	quiet = TRUE
+	max_combat_health = 4
+	special_attack_type = SPECIAL_ATTACK_OTHER
+	special_attack_type_message = "has a lower cooldown than normal special moves and deals damage."
+	special_attack_cry = "*wave"
+
+/obj/item/toy/prize/reticence/super_special_attack(obj/item/toy/prize/victim) 
+	special_attack_cooldown-- //Has a lower cooldown...
+	victim.special_attack_cooldown++ //and increases the opponent's cooldown by 1...
```

Comment:
this isn't mentioned in the description of this special attack

Reply:
you right, i buffed it and forgot to change it